bravado/fido_client.py

```diff
@@ -2,29 +2,26 @@
 # -*- coding: utf-8 -*-
 
 #
-# Copyright (c) 2014, Yelp, Inc.
+# Copyright (c) 2015, Yelp, Inc.
 #
 
-"""Asynchronous HTTP client abstractions.
-"""
-
 import logging
 
-import fido
 from yelp_uri import urllib_utf8
 
-from bravado import http_client
+from bravado.fido_future import FidoFuture
+from bravado.mapping.http_client import APP_FORM, HttpClient
 from bravado.multipart_response import create_multipart_content
 from bravado.mapping.param import stringify_body as param_stringify_body
 
 log = logging.getLogger(__name__)
 
 
-class AsynchronousHttpClient(http_client.HttpClient):
-    """Asynchronous HTTP client implementation.
+class FidoClient(HttpClient):
+    """Fido HTTP client implementation.
     """
 
-    def start_request(self, request_params):
+    def request(self, request_params, op=None):
```

Comment:
This is because `op` is not required for api-docs fetch. For that instance, `client.request(request_params)` has to be called without `op`.